DMTN-306.tex

```diff
@@ -1,79 +1,119 @@
+
+
+\subsection{From archive to data access centers}
+\label{summit-to-data-access-centers}
+
+Annually-released data products must be distributed to approximately 15 to 20 data access centers across the America, Europe, and Asia-Pacific regions, where scientific analysis will be conducted. These distribution campaigns will be centrally coordinated by Rubin to ensure timely delivery of data releases to all analysis centers. The goal is to distribute the multi-petabyte datasets across multiple sites using the network bandwidth available at several participating sites, thereby reducing the load on the archive center \cite{RTN-086}.
+
+\section{Data movement tools}
+\label{section-data-movement-tools}
+
+Several software tools are employed to implement the use cases outlined in the previous section. CERN's Rucio \cite{rucio2019} and its companion FTS \cite{FTS} manage the movement of files between the archive site and data facilities, as well as from the archive to the data access centers. In addition, Rubin-specific tools have been developed to register files and automate actions when replicated files arrive at their destination. These tools and their usage are described in the following subsections.
+
+Rubin Observatory operates a dedicated instance of Rucio, configured to transfer files between the Rucio Storage Elements (RSEs) at each facility. These storage endpoints support a data movement protocol that Rucio utilizes to transport data across them. The US and UK data facilities use XrootD \cite{xrootd}, while the French data facility uses dCache \cite{dCache}. All of these systems expose the webDAV protocol \cite{webdav}, an extension of HTTP \cite{http1.1}. Data is transferred securely across sites using confidential channels built on top of secure HTTP.
+
+Each processing facility exposes at least two RSEs: one for storing input data required for processing (e.g., raw images, calibration data, reference catalogs, etc.) and another endpoint for storing the products generated by the image processing pipelines. All RSEs are configured to use the identity logical-to-physical filename mapping. This configuration ensures that the file pathnames are preserved relative to the Butler repository's datastore location, which is critical for proper file replication to the destination where the Butler expects to find them.
+
+\subsection{Registration of files to replicate}
+\label{registration}
+
+To perform replication, we create Rucio Datasets, each composed of a set of files that are already in their appropriate locations at the source RSE. Upon registration, pre-configured Rucio subscriptions trigger the actual file movement to the destination facility, in accordance with the defined replication rules. Rucio delegates the data transfer to FTS, which then instructs the storage endpoints at the facilities to move the data, typically by requesting the destination facility to pull the data from the source facility.
```

Comment:
Maybe add this:

The use of Datasets allows grouping of related files, use of subscription patterns applied to spatially-defined Dataset names to associate spatial regions with RSEs, and a clear way to know when all related files have been generated (via Dataset closure) and replicated.

Reply:
Added.